trade_remedies_public/config/settings/hardening.py

```diff
@@ -44,22 +44,29 @@
 # regular expression listed in SECURE_REDIRECT_EXEMPT
 SECURE_SSL_REDIRECT = True
 
-SECURE_HSTS_SECONDS = 0
+SECURE_HSTS_SECONDS = 86400  # 1 day
```

Comment:
I think we should change this to 60 seconds or something small so we can test it doesn't break anything on UAT before moving it to a higher value like a day or a week. If we put a day and there is a bug, we will have to wait a day to retest.

```suggestion
SECURE_HSTS_SECONDS = 60  # 60 seconds
```